docs/source/examples/substrafl/get_started/run_mnist_torch.ipynb

```diff
@@ -0,0 +1,797 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    dependencies = Dependency(pypi_dependencies=["numpy==1.23.1", "torch==1.11.0", "scikit-learn==1.1.1"])
</pre><p>1.11? maybe we want to update to 1.13.1 (at least until we bump it to 2+)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>